.vscode/settings.json

```diff
@@ -5,4 +5,16 @@
   ],
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
+  "sqltools.connections": [
+    {
+      "previewLimit": 50,
+      "server": "localhost",
+      "port": 5432,
+      "driver": "PostgreSQL",
+      "name": "docker-traction",
+      "username": "postgres",
+      "password": "postgresPass",
+      "database": "traction"
+    }
+  ],
```

Comment:
Don't think this was to be comitted?

Reply:
It's in the settings file and it's still valid for other users. If we don't want this in the repo then we should un-version the file.

I can remove it if desired.

Reply:
I'd clear it if it's dev machine specific tooling

Reply:
added to ignore, but this change was already in an earlier commit.